tests/backends/base/test_creation.py

```diff
@@ -175,3 +177,14 @@ def test_circular_reference_with_natural_key(self):
         obj_b = CircularB.objects.get()
         self.assertEqual(obj_a.obj, obj_b)
         self.assertEqual(obj_b.obj, obj_a)
+
+    def test_serialize_db_to_string_base_manager(self):
+        SchoolClass.objects.create(year=1000, last_updated=datetime.datetime.now())
```

Comment:
Minor but I think a warning will be raised here as this is passing a naive datetime will timezone support is enabled.

Reply:
Timezone support is disable by default and we use naive datetimes in many tests. I think we can leave it.